llvm/test/CodeGen/NVPTX/LoadStoreVectorizer.ll

```diff
@@ -1,8 +1,8 @@
 ; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
-; RUN: llc < %s | FileCheck -check-prefix=ENABLED %s
-; RUN: llc -disable-nvptx-load-store-vectorizer < %s | FileCheck -check-prefix=DISABLED %s
-; RUN: %if ptxas %{ llc < %s | %ptxas-verify %}
-; RUN: %if ptxas %{ llc -disable-nvptx-load-store-vectorizer < %s | %ptxas-verify %}
+; RUN: llc -march=nvptx64 -mcpu=sm_90 -mattr=+ptx88 < %s | FileCheck -check-prefix=ENABLED %s
+; RUN: llc -march=nvptx64 -mcpu=sm_90 -mattr=+ptx88 -disable-nvptx-load-store-vectorizer < %s | FileCheck -check-prefix=DISABLED %s
+; RUN: %if ptxas-sm_90 && ptxas-isa-8.8 %{ llc -march=nvptx64 -mcpu=sm_90 -mattr=+ptx88 < %s | %ptxas-verify -arch=sm_90 %}
+; RUN: %if ptxas-sm_90 && ptxas-isa-8.8 %{ llc -march=nvptx64 -mcpu=sm_90 -mattr=+ptx88 -disable-nvptx-load-store-vectorizer < %s | %ptxas-verify -arch=sm_90 %}
```

Comment:
These should be left with the defaults.

Pragma's own tests should live in a separate test. They are coincidental here.

Reply:
Good point. I've removed the version specifiers and with the addition of the new gate all the pragmas as no longer generated, as we expect.